### ISSUE TITLE
Audio only

### ISSUE THREAD

**Issue by W1abc** [ilucasgoncalves/VAVL#6]
How do you run the implementation when the dataset has only audio or only video files？

**Comment by ilucasgoncalves** [comment]
One way is to quickly modify the train loop to run only the audio or video part. e.g. for audio:


    for epoch in range(epochs):
        print("Epoch:",epoch)
        lm.init_stat()
        for xy_pair in tqdm(total_dataloader["train"]):
            modelWrapper.set_train()
            xa = xy_pair[0]
            y = xy_pair[1]
            mask = xy_pair[2]
            
            xa=xa.cuda(non_blocking=True).float()
            y=y.cuda(non_blocking=True).float()
            mask=mask.cuda(non_blocking=True).float()

            with autocast():

                preds_a, x_in, rec_pred = modelWrapper.feed_forward(xa, None, mode = 'acoustic', attention_mask=mask)
                loss_rec_a = utils.MSE_loss(rec_pred, x_in)

                total_loss_a = 0.0
                if args.label_type == "dimensional":
                    ccc = utils.CCC_loss(preds_a, y)
                    loss = 1.0-ccc
                    total_loss_a += loss[0] + loss[1] + loss[2] + loss_rec_a

                elif args.label_type == "categorical":
                    total_loss_a += utils.CE_category(preds_a, y) + 2*loss_rec_a

            ## Backpropagation
            modelWrapper.backprop(total_loss_a, 'acoustic')


Do the same for dev and test only use these parts and set the forward calling to acoustic like: `preds_a, _, _= modelWrapper.feed_forward(xa, None, mode = 'acoustic', attention_mask=mask)`

Additionally, if you will be only loading audio or only video. The dataloader will have to be modified to your needs.
